crawler.py

```diff
@@ -39,11 +39,11 @@
 CHROME = 'chrome'
 FIREFOX = 'firefox'
 
-MAJESTIC_URL = "http://downloads.majesticseo.com/majestic_million.csv"
-WEEK_IN_SECONDS = 604800
 RESTART_RETRIES = 5
 
-OUR_PYFUNCEBLE_CONFIG = {"share_logs": False}
+TRANCO = Tranco(cache=True, cache_dir='.tranco')
+TRANCO_LIST = TRANCO.list()
```

Comment:
Why don't we fetch the list inside `get_domain_list()`? (Why fetch the list if we run crawler.py with `--help` or something?)